lib/user/getPlayersThumbnail.js

```diff
@@ -0,0 +1,47 @@
+// Includes
+var http = require('../util/http.js').func
```

Comment:
Please use `const` for imports that don't change

Reply:
Alright